src/app/screens/backupWallet/index.tsx

```diff
@@ -58,14 +57,14 @@ const TransparentButtonContainer = styled.div((props) => ({
 function BackupWallet(): JSX.Element {
   const { t } = useTranslation('translation', { keyPrefix: 'BACKUP_WALLET_SCREEN' });
   const navigate = useNavigate();
-  const { seedPhrase } = useWalletSelector();
-  const { storeSeedPhrase } = useWalletReducer();
+  const { initSeedVault, storeSeed } = useSecretKey();
 
   useEffect(() => {
-    if (!seedPhrase) {
+    (async () => {
+      await initSeedVault('');
       const newSeedPhrase = generateMnemonic();
-      storeSeedPhrase(newSeedPhrase);
-    }
+      await storeSeed(newSeedPhrase);
+    })();
```

Comment:
the behaviour changed here. need to check why 
```
if (!seedPhrase) {
}
```

is no longer required

Reply:
it was no longer necessary since the seedphrase can no longer be synced between tabs and we removed forward/backward capabilities

Reply:
This seems rally dangerous 👀 If we introduce a bug at some point and accidentally call `await storeSeed(newSeedPhrase);` the seedvault will overwrite the old one without issue because it doesn't check if there isn't already a stored seed. I think we need ot throw an error if we try to set a seed when one already exists and then rethink this flow a little.